4nvOC_watch

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# 4nvOC_watch to monitor nvOC running scripts
+
+v0001="papampi; Initial release"
+
+source ${NVOC}/1bash
+nvOC_4nvOC_watch_Dev="0001"
+nvOC_4nvOC_watch_ver="$nvOC_ver.$nvOC_4nvOC_watch_Dev"                
+
+echo "Starting nvOC Watch to monitor nvOC scripts in 60 seconds"
+sleep 60
+
+# Infinite Loop
+while true
+do
+  source ${NVOC}/1bash
+  echo "$(date) Check nvOC scripts"
+
+  if ! ps ax | grep -q "[3]main"
+  then
+    echo "3main is not running, checking 2unix"
+    if ! ps ax | grep -q "[2]unix"
+    then
+      echo "$(date) - 2unix stopped running, restarting it"  | tee -a  ${NVOC}/nvoc_logs/5_watchdoglog
+      bash ${NVOC}/bash nvOC restart
```

Comment:
This should be bash ${NVOC}/nvOC restart. But you should take care of checking if it is a gnome-terminal install or a service install in order to perform the right operation. Then you have to exit from this script.

Reply:
Hmmm... 
Is it enought to check for /etc/systemd/system/nvoc.service ?

Reply:
Better to check if it is "enabled" with
if sudo systemctl is-enabled nvoc.service; then ...
But I have to add the [Install section in the service file otherwise it is not getting enabled, and it will always return "static" (exit code: 0) so the if statement is always returning true